Formula/prototool.rb

```diff
@@ -0,0 +1,31 @@
+class Prototool < Formula
+  desc "Your Swiss Army Knife for Protocol Buffers"
+  homepage "https://github.com/uber/prototool"
+  url "https://github.com/uber/prototool/archive/v1.1.0.tar.gz"
+  sha256 "7759bd4db9a907d3dea4abb34ff18ddbc814f4de56d43aca8d2f402e6fd623fa"
+
+  depends_on "automake" => :build
```

Comment:
What is `automake` being used for here? It's a pretty unusual dependency when also using `go`.

Reply:
I think I was overzealous and was thinking that I depended on `make` below, which is installed from `xcode-select` - I'm assuming we can assume users have installed command line tools, is that correct? Removing this line, let me know otherwise.